.infrastructure/helm-chart/todoapp/templates/pvc.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Chart.Name }}-pvc
+  namespace: {{ .Values.todoapp.namespace }}
+spec:
+  volumeName: {{ .Chart.Name }}-pv
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{ .Values.todoapp.pvcStorageRequest}}
```

Comment:
Ensure that the `todoapp.pvcStorageRequest` value is defined in the `values.yaml` file. This is necessary for specifying the storage request for the PersistentVolumeClaim.